share/revng/valgrind.supp

```diff
@@ -0,0 +1,36 @@
+# This file is distributed under the MIT License. See LICENSE.md for details.
+{
+   Cond on ParseCommandLineOptions
+   Memcheck:Cond
+   ...
+   fun:_ZN4llvm2cl23ParseCommandLineOptionsEiPKPKcNS_9StringRefEPNS_11raw_ostreamES2_b
+   ...
+}
+{
+   Value8 on ParseCommandLineOptions
+   Memcheck:Value8
+   ...
+   fun:_ZN4llvm2cl23ParseCommandLineOptionsEiPKPKcNS_9StringRefEPNS_11raw_ostreamES2_b
+   ...
+}
+{
+   Addr1 on ParseCommandLineOptions
+   Memcheck:Addr1
+   ...
+   fun:_ZN4llvm2cl23ParseCommandLineOptionsEiPKPKcNS_9StringRefEPNS_11raw_ostreamES2_b
+   ...
+}
```

Comment:
I ran (the then-equivalent) `revng daemon-self-test` with valgrind and noted the complaints

Reply:
Mmmh, this is hiding defects, I'm not sure we want this.
I'm in favor of suppressing something only if it's a false positive, otherwise I prefer to keep showing up.

Reply:
Dropped, couldn't make them show up in current revng